modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -140,6 +141,8 @@ public AbstractTypeScriptClientCodegen() {
         instantiationTypes.put("array", "Array");
 
         typeMapping = new HashMap<String, String>();
+        typeMapping.put("Set", "Set");
+        typeMapping.put("set", "Set");
```

Comment:
FYI: this line broke `typescript-angular` as described in https://github.com/OpenAPITools/openapi-generator/issues/8258#issuecomment-755409548.

More generally, it broke all the typescript generators that (for a response) just return the JSON object provided by a HTTP backend (which don't have runtime information on which JSON arrays correspond to schemas that have `uniqueItems` set). `Set<T>` has functions in it's prototype that `Array<T>` doesn't have (`add`, `clear`, `delete`, ...) and vice versa (`push`, `pop`, ...) so they are not interchangeable